src/Randomizer.Multiplayer.Server/README.md

```diff
@@ -0,0 +1,95 @@
+# SMZ3 Cas’ Multiplayer Server
+
+Randomizer.Multiplayer.Server is a cross platform SignalR .NET application designed to route messages between various players in a Cas' Multiplayer game. In this implementation, all seed and rom generation is handled on the client end and the server simply acts a sort of router for the traffic between the players in a game.
+
+## Prerequisites
+
+- ASP.NET Core 6
+- .NET Core 6
+- IIS, Nginx, or similar to use as a proxy (recommended)
+- Visual Studio 2022 (for building)
+
+## Creating a new Build
+
+1. Update the package version
+1. Publish the Randomizer.Multiplayer.Server project to the bin\Release\net6.0\publish\ folder
+1. Execute MultiplayerServerZipper.ps1 in the setup folder to create a versioned zip file with
+
+## appsettings.json
+
+To run the server, you will need to execute to have a configured appsettings.json file in the working directory from where you run the server executable. You can find an example of the appsettings [here](appsettings.example.json).
+
+Below are explanations of all of the settings:
+
+- SMZ3
+    - **ServerUrl**: Required. Specifies the base URL that is used for generating game URLs to send to players. Include the protocol and subdomain (e.g. https<nolink>://smz3.whateverdomain.com)
```

Comment:
It's generally better to use `example.com` for examples to avoid the situation where someone buys "whateverdomain.com" and does something shady with it.